infra/aws/terraform/management-account/organization-accounts-boskos.tf

```diff
@@ -41,3 +40,167 @@ module "k8s_infra_e2e_boskos_003" {
   iam_user_access_to_billing = "ALLOW"
   parent_id                  = aws_organizations_organizational_unit.boskos.id
 }
+
+##############################################
+# Accounts used by boskos in the EKS cluster #
+##############################################
+
+module "k8s_infra_e2e_boskos_004" {
```

Comment:
I'm not sure:

* what are accounts 001 - 003 used for (I assume for boskos running in GKE cluster, but I don't find any references to those accounts)
* do we want to use different numbering for accounts used in EKS cluster, e.g. 100 - 110

Reply:
Done.